src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -1043,22 +1045,26 @@ function ScatterplotViz(props: VisualizationProps<Options>) {
       ? ({
           entityId: computedYAxisDetails?.entityId,
           variableId: computedYAxisDetails?.variableId,
-          displayName: data.value?.computedVariableMetadata?.displayName?.[0],
+          displayName: 'foo', // data.value?.computedVariableMetadata?.displayName?.[0],
```

Comment:
Not sure what this part of the code does.

The back end doesn't seem to return `computedVariableMetadata.displayName`.

Boxplot has the same code.  I will investigate!

Reply:
Idk if it's relevant yet, but some computed variables we will provide display names for. For ex in beta div the two axes of the scatter plot have display names that depend on the results of the compute.

Reply:
Turned out that the `displayName` prop was off-spec and unused. I removed it.